test/data/test_multi_nested_tensor.py

```diff
@@ -81,6 +81,18 @@ def test_multi_nested_tensor_basics(device):
     assert dense_multi_nested_tensor.shape == (multi_nested_tensor.shape[:-1] +
                                                (max_len, ))
 
+    multi_nested_tensor_with_nan = multi_nested_tensor.clone()
+    # Set some values to be na
+    multi_nested_tensor_with_nan.values[0::2] = -1
+
+    # Test fill_col
+    multi_nested_tensor_1 = multi_nested_tensor_with_nan.clone()
+    for col in range(multi_nested_tensor_with_nan.num_cols):
+        multi_nested_tensor_1.fillna_col(col, 100)
+    assert not torch.all(multi_nested_tensor_1.values == -1).any()
+    assert torch.all(multi_nested_tensor_1.values[
+        multi_nested_tensor_with_nan.values.isnan()] == 100)
```

Comment:
Same here. Test for column-specific fillna

Reply:
Cool. I changed the code. Now it fills different columns with different fill values and checks the correctness for each column.